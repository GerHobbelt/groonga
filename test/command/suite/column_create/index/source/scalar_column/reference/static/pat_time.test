table_create Times TABLE_PAT_KEY Time

table_create Data TABLE_NO_KEY
column_create Data time COLUMN_SCALAR Times

load --table Data
[
{"time": "2022-03-15T00:00:00"}
]

column_create Times data_time COLUMN_INDEX Data time

index_column_diff Times data_time
