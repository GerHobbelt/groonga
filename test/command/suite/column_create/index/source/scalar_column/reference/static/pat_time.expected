table_create Times TABLE_PAT_KEY Time
[[0,0.0,0.0],true]
table_create Data TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Data time COLUMN_SCALAR Times
[[0,0.0,0.0],true]
load --table Data
[
{"time": "2022-03-15T00:00:00"}
]
[[0,0.0,0.0],1]
column_create Times data_time COLUMN_INDEX Data time
[[0,0.0,0.0],true]
index_column_diff Times data_time
[[0,0.0,0.0],[]]
